backend/ops_api/ops/utils/change_requests.py

```diff
@@ -28,13 +28,17 @@ def create_notification_of_new_request_to_reviewer(change_request: ChangeRequest
         division_director_ids.add(division.deputy_division_director_id)
     fe_url = current_app.config.get("OPS_FRONTEND_URL")
 
-    approve_url = f"{fe_url}/agreements/approve/{agreement_id}"
+    approve_url = (
+        f"{fe_url}/agreements/approve/{agreement_id}?type=status-change"
+        if change_request.has_status_change
```

Comment:
@stevtek we'll need to figure out how to account for the Status-Change-To type

Reply:
You could extract the base part of the URL out ({fe_url}/agreements/approve/{agreement_id}?type=) and then just have an if-else that handles the choosing of the type here instead of doing a ternary potentially. If I'm understanding what you're saying above, at least. It might be worth doing regardless considering having the base URL repeated makes it more prone to accidental typos causing bugs.

Reply:
⭐thanks @rajohnson90 for your help with this today